workers/dockerfiles/Dockerfile.no_op

```diff
@@ -20,8 +33,16 @@ COPY workers/data_refinery_workers/downloaders/requirements.txt .
 
 RUN pip3 install --upgrade pip
 
+# Noop-specific requirements
+ENV R_LIBS "/usr/local/lib/R/site-library"
+COPY workers/gene_indexes gene_indexes/
+COPY workers/install_gene_convert.R .
+RUN Rscript install_gene_convert.R
+# End Noop-specific
```

Comment:
Do these commands have any python dependencies? If not it might be nice to move it above line 32 so adding a python dependency doesn't require this step to be rerun.

Reply:
Done